docs/checks/internal/checks.yaml

```diff
@@ -691,14 +691,20 @@ checks:
     description: |
       Risk: `High`  (known vulnerabilities)
 
-      This check determines whether the project has open, unfixed vulnerabilities
-      using the [OSV (Open Source Vulnerabilities)](https://osv.dev/) service. An open
-      vulnerability is readily exploited by attackers and should be fixed as soon as
+      This check determines whether the project has open, unfixed vulnerabilities 
+      in it's dependencies using the [OSV (Open Source Vulnerabilities)](https://osv.dev/) service.
+      An open vulnerability is readily exploited by attackers and should be fixed as soon as
       possible.
     remediation:
       - >-
         Fix the vulnerabilities. The details of each vulnerability can be found
```

Comment:
maybe add or "update your dependencies to a non-vulnerable version"

Reply:
Yeah, need to include remediation tips for vulnerable dependencies. Something like: "If the vulnerability is in a dependency, update the dependency to a non-vulnerable version. If no update is available, consider whether to remove the dependency."